pmlc/dialect/pxa/transforms/vectorize_mem.cc

```diff
@@ -295,9 +300,9 @@ struct VectorizeMemImpl {
         VectorType::get({memOpsPlan.subgroupSize * tileSize},
                         vectorReduce.getVectorType().getElementType());
 
-    auto nemMemrefType =
+    auto memMemrefType =
```

Comment:
This should probably be renamed to `newMemrefType`

Reply:
you are right! let me change it